server/main.lua

```diff
@@ -291,13 +333,15 @@ QBCore.Functions.CreateCallback('qb-garage:server:GetPlayerVehicles', function(s
                     v.state = Lang:t("status.impound")
                 end
 
-                local fullname
+                local fullname = "Unknown"
```

Comment:
```suggestion
                local fullname
```
This value isn't used.